compiler/rustc_mir_build/src/thir/pattern/usefulness.rs

```diff
@@ -1017,10 +1155,18 @@ crate struct MatchArm<'p, 'tcx> {
     crate has_guard: bool,
 }
 
+#[derive(Clone, Debug)]
+crate enum Reachability {
+    /// Potentially carries a set of sub-branches that have been found to be unreachable. Used only
+    /// in the presence of or-patterns, otherwise it stays empty.
+    Reachable(Vec<Span>),
```

Comment:
This comment is slightly confusing, because it says "that have found to be unreachable", but the variant is called `Reachable`.